optimizely/config/datafileprojectconfig/entities/entities.go

```diff
@@ -51,6 +55,20 @@ type FeatureFlag struct {
 	Variables     []Variable `json:"variables"`
 }
 
+// GetVariable returns variable for key from feature flag
+func (f *FeatureFlag) GetVariable(key string) (Variable, error) {
```

Comment:
Instead of adding this functionality to the model, let's add this method to the project config itself to get a variable with key from the feature